src/commands.js

```diff
@@ -114,47 +116,45 @@ export const help = msg => {
 };
 
 // Send details of a given command
-export const helpSpecific = msg => {
-  const { content, channel } = msg;
-  const userRoles = msg.member.roles;
-  const command = content.split(' ')[1];
+export const helpSpecific = (msg, command) => {
+  // const userRoles = msg.member.roles;
   let commandInfo;
 
   // True if command exists, otherwise returns false
   const commandExists =
     commandsList.find(el => {
-      if (Array.isArray(el.names)) {
-        if (el.names.find(name => command === name) !== undefined) {
-          commandInfo = {
-            title: '`' + el.names.join('` | `') + '`',
-            description: el.description,
-            role: el.role,
-          };
-          return true;
-        }
-        return false;
-      }
-      if (el.names === command) {
+      if (el.names.find(name => command === name) !== undefined) {
```

Comment:
We can actually shorten this with `if (!el.names.find(name => command === name)) `